### DISCUSSION TITLE
About the RNA04 dataset to train the model

### DISCUSSION THREAD

**Discussion by charlesxu90** [GoekeLab/m6anet#144] [Q&A] [answered]
Dear @jonathangoeke @chrishendra93 @yuukiiwa @kristinrma ,

Thanks for the nice work. In your recent release, you mentioned that RNA004 is supported by m6anet. I think it was very interesting and hope to use the same dataset for another development task.

Could you please share the link to the RNA004 dataset that you train the model on? I searched online, but cannot find it. I believe RNA004 will be the new standard for direct RNA sequence, it has different data distributions so developing methods based on RNA004 should be a must.

Thanks a lot for the nice work.

Best regards，

**Comment by jonathangoeke** [comment, marked as answer]
Hi @charlesxu90, thanks for reaching out. These data were generated through the early access program from Oxford Nanopore Technologies, and we are not allowed to release any of the raw data as these are from pre-release versions. We will try to generate RNA004 data from final RNA004 release, wich we will be able to share then. We will post an update here

**Reply by charlesxu90** [reply]
Get it. Thanks for the reply.

**Reply by BrendanBeahan** [reply]
Hi @jonathangoeke I'm terribly sorry if this is being a bit pushy, but has there been any update on the availability of this dataset? Would be very excited to utilize it.

Kind regards,
Brendan
